base/base/Decimal_fwd.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <base/types.h>
+
+namespace wide
+{
+
+template <size_t Bits, typename Signed>
+class integer;
+
+}
+
+using Int128 = wide::integer<128, signed>;
```

Comment:
maybe replace with `#include <Core/Types_fwd.h>`?

Reply:
I meant lines 5-16 in this file repeat the same declarations that exist in `Types_fwd.h`